deploy/helm/sumologic/values.yaml

```diff
@@ -8,6 +8,8 @@ nameOverride: ""
 deployment:
   nodeSelector: {}
   tolerations: {}
+  affinity: {}
+  podAntiAffinity: "soft"
```

Comment:
can you also leave a comment here for the customers that the accepted values are "soft" and "hard", and what those mean?